api/problem/create.py

```
@@ -0,0 +1,40 @@
+    if not (SUG.THR.DBV and SUG.THR.UDT["Urole"] >= SUG.PER["problem"]["create"]): return SUG.REQ.RES.error(3)
+    # DECLARATION -> QUERY
+    SUG.REQ.RES.write(database.request(
+        "INSERT INTO PROBLEMS (Uid, Kid, Pedited, Pmeta, Psolution, Pdataen, Pdataes, Pdatade) VALUES (?, ?, ?, ?, ?, ?, ?, ?)",
+        [
+            SUG.THR.UDT["Uid"],
+            SUG.REQ.PST["Kid"],
+            time.now(),
+            SUG.REQ.PST["Pmeta"],
+            SUG.REQ.PST["Psolution"],
+            SUG.REQ.PST["Pdataen"],
+            SUG.REQ.PST["Pdataes"] if SUG.REQ.PST["Pdataen"] else None,
```

Ruff (0.11.9)

`SUG` may be undefined, or defined from star imports

(F405)